packages/js/data/src/product-form/types.ts

```diff
@@ -23,10 +23,13 @@ export type ProductFormSection = BaseComponent & {
 
 export type Subsection = BaseComponent;
 
+export type Tabs = BaseComponent; // TODO add properties here?
```

Comment:
I believe we'll at least want a `name` and `title` here, as Louren's [is adding in the slot fills.](https://github.com/woocommerce/woocommerce/pull/36551/files#r1083818216)